base/server/python/pki/server/cli/acme.py

```diff
@@ -1018,8 +1068,26 @@ def execute(self, argv):
         issuer_conf = os.path.join(acme_conf_dir, 'issuer.conf')
         config = {}
 
-        logger.info('Loading %s', issuer_conf)
-        pki.util.load_properties(issuer_conf, config)
+        if issuer_type:
+            source = '/usr/share/pki/acme/issuer/%s/issuer.conf' % issuer_type
+        else:
+            source = issuer_conf
+
+        logger.info('Loading %s', source)
+        pki.util.load_properties(source, config)
+
+        # use silent mode if --type or -D was specified
+        if issuer_type or props:
+
+            logger.info('Setting properties:')
+            for name, value in props.items():
+                logger.info('- %s: %s', name, value)
+                pki.util.set_property(config, name, value)
+
+            pki.util.store_properties(issuer_conf, config)
```

Comment:
Are we trying to load from `source` (in line 1077) and copy to `issuer_conf`?

Reply:
Yes, if the `--type` is specified, it will load the issuer.conf/database.conf template from /usr/share/pki/acme. This is mainly used for setting up a new ACME responder.

If the `--type` is not specified, will load the current issuer.conf/database.conf from the instance configuration folder. This is mainly used for modifying the existing configuration.

In both cases it will update the issuer.conf/database.conf in the instance configuration folder. I added some comments in the code.